src/constants.ts

```diff
@@ -1,3 +1,8 @@
+/** @internal */
+export type BSON_MAJOR_VERSION = 5;
+/** @internal */
+export const BSON_MAJOR_VERSION: BSON_MAJOR_VERSION = 5;
```

Comment:
why do we need the type and const gymnastics here? more specifically, why do we need the explicit `type` export for this?

Reply:
This made more sense when we had the many getters approach and I wanted TS to check the return type strictly. fixed!